release-calendar/src/app.ts

```diff
@@ -27,9 +27,53 @@ async function main() {
     entities: [Release],
     synchronize: !JSON.parse(process.env.PRODUCTION), //recreate database schema on connect
     logging: false,
-  }).catch(error => {
-    throw error;
-  }) as Connection;
+  })
+    .then(async connection => {
```

Comment:
You don't need a `.then` with the `await` operator. You can do

```suggestion
    const connection = await createConnection( . . . );
    const releaseRepo = connection.getRepository(Release);
```